scripts/xtp_modify_jobfile.in

```diff
@@ -0,0 +1,191 @@
+#! /usr/bin/env python3
+#
+# Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+VERSION = '@PROJECT_VERSION@ #CSG_GIT_ID#'
+import sys
+import argparse
+import re
+import lxml.etree as lxml
+import fnmatch
+
+
+PROGTITLE = 'THE VOTCA::XTP MODIFY JOBFILE'
+PROGDESCR = 'Creates a subset Jobfile from a larger Jobfile using selection criteria'
+VOTCAHEADER = '''\
+==================================================
+========   VOTCA (http://www.votca.org)   ========
+==================================================
+
+{progtitle}
+
+please submit bugs to @PROJECT_CONTACT@
+xtp_update_mapfile, version {version}
+
+'''.format(version=VERSION, progtitle=PROGTITLE)
+
+
+def okquit(what=''):
+    if what != '':
+        print(what)
+    sys.exit(0)
+# =============================================================================
+# PROGRAM OPTIONS
+# =============================================================================
+
+
+class XtpHelpFormatter(argparse.HelpFormatter):
+    def _format_usage(self, usage, action, group, prefix):
+        return VOTCAHEADER
+
+
+progargs = argparse.ArgumentParser(prog='xtp_modify_jobfile',
+                                   formatter_class=lambda prog: XtpHelpFormatter(
+                                       prog, max_help_position=70),
+                                   description=PROGDESCR)
+
+progargs.add_argument('-i', '--input',
+                      dest='j_input',
+                      action='store',
```

Comment:
`store` is the default action, so you don't need to add it explicitely

Reply:
copied it from the other scripts, maybe we should remove it everywhere then